shorts_pipeline/modules/video_assembler.py

```
@@ -0,0 +1,56 @@
+        for video, script in rows:
+            try:
+                footage = self._fetch_footage(script.idea, script.id)
+                vc = VideoFileClip(str(footage)).resize((self.cfg["video"]["width"], self.cfg["video"]["height"]))
+                ac = AudioFileClip(video.audio_path)
+                clip = vc.subclip(0, min(vc.duration, ac.duration + 1)).set_audio(ac)
+                sub = self._subtitle_clip(f"{script.hook} {script.retention_hook}", clip.duration)
+                final = CompositeVideoClip([clip, sub])
+                out = Path("output/videos") / f"video_{script.id}.mp4"
+                tmp = out.with_suffix(".tmp.mp4")
+                final.write_videofile(str(tmp), fps=self.cfg["video"]["fps"], codec="libx264", audio_codec="aac", logger=None)
+                subprocess.run(["ffmpeg", "-y", "-i", str(tmp), "-c:v", "h264_nvenc", "-preset", self.cfg["video"]["nvenc_preset"], "-c:a", "aac", str(out)], check=True)
```

Ruff (0.15.20)

[error] `subprocess` call: check for execution of untrusted input

(S603)